### DISCUSSION TITLE
It's not the same as the ripple effect in Foundation

### DISCUSSION THREAD

**Discussion by for-the-zero** [material-components/material-web#5530] [Q&A]
[in the Pressed section in States](https://m3.material.io/foundations/interaction/states/applying-states#d561ace2-b8e9-4a8c-bee7-8cdeb6878160), it looks like this

![](https://lh3.googleusercontent.com/6T0GD3aYYU4ANr1XUkAATDGxOSm0_ua2hmVSfwS_I6oBFfoFMjvw2f3NpcwkXtNRVX8JJufJCaXMCO_e8SDGzScMXdy1gMLgRsWumKC7PSHF=s0)

This effect is very similar to Material 2's

But the ripple effect is new in Android 12 and above and in this project, why is this

**Comment by asyncliz** [comment]
I think this is just an incorrect image on material.io. The main difference between M2 and M3's ripples are:

- M3 ripples have a soft edge
- M3 ripples don't show up while focusing

There's a "Feedback" link at the bottom of material.io. I'd suggest filing some feedback for the design team to address it!
